assets/src/edit-story/components/checklist/checks/pageBackgroundLowTextContrast/check.js

```diff
@@ -0,0 +1,427 @@
+/*
```

Comment:
that's a lot of code!!! 😓 this was mostly ripped out of the accessibility check, but I modified it and wrote more comments because it was pretty hard to tell what was going on sometimes. Sorry y'all.

Reply:
no no, extra documentation is always nice! Thanks for adding more comments